src/plugins/footer-links/package.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
@bradenmacdonald if having a package.json for a `/plugins` directory in your Course Authoring plugins, I'm assuming this isn't needed any longer now that FPF would allow for the components to be plugged in using `DirectPlugin`. Just wanted to get a confirmation that this is probably a remnant of a different plugin approach.

Reply:
@jsnwesson The pluggable "Pages & Resources" section of Course Authoring uses a different plugin approach which still uses package.json files for each plugin. But I don't think it's needed here, nor with FPF in general.

Reply:
Yes, this is no longer needed now as I am importing FooterLink from path.